collections/_article/dev-snapshot-godot-4-5-beta-1.md

```diff
@@ -0,0 +1,364 @@
+- Fix LightmapGI shadow leaks. ([GH-107254](https://github.com/godotengine/godot/pull/107254))
+
+### Breaking changes
+We try to minimize breaking changes, but sometimes they are necessary in order to fix high priority issues. Where we do break compatibility, we do our best to make sure that the changes are minimal and require few changes in user projects.
+
+You can find a list of such issues by filtering the merged PRs in the 4.5 milestone with the [`breaks compat` label](https://github.com/godotengine/godot/issues?q=milestone%3A4.5%20is%3Amerged%20label%3A%22breaks%20compat%22). Here are some which are worth being aware of:
+- Tilemap physics are now handled in chunks. Only affects `get_coords_for_body_rid`, as now a single body can cover multiple cells. ([GH-102662](https://github.com/godotengine/godot/pull/102662))
+- Internal nodes are no longer duplicated. Only affects users deliberately utilizing internal nodes. ([GH-89442](https://github.com/godotengine/godot/pull/89442))
+- `NodeOneShot` fading now uses self delta instead of input delta. Brings behavior closer to other `AnimationNode`s, as the old implementation was exclusive to `NodeOneShot`. ([GH-101792](https://github.com/godotengine/godot/pull/101792))
+- `TreeItem`'s `alt_text` suffixes renamed to `description`. This brings the naming scheme in-line with the Accessibility API. Only relevant for those already using 4.5 dev versions. ([GH-107540](https://github.com/godotengine/godot/pull/107540))
+- 2D & 3D Navigation region and link updates are now asynchronous. (2D: [GH-107381](https://github.com/godotengine/godot/pull/107381), 3D: [GH-106670](https://github.com/godotengine/godot/pull/106670))
+- `NavigationServer2D` avoidance callbacks changed from `Vector3` to `Vector2`. ([GH-107256](https://github.com/godotengine/godot/pull/107256))
+- `CONTEXT_SLOT_FILESYSTEM_CREATE` will now always use the base directory. No longer recieves a full path when called. ([GH-107085](https://github.com/godotengine/godot/pull/107085))
+- Removed the `gradle_build`/`compress_native_libraries` export option. With Android builds now supporting 16kb pages, the native libraries are now required to be uncompressed. ([GH-106359](https://github.com/godotengine/godot/pull/106359))
+- "Areas Detect Static Bodies" setting removed from Jolt Physics. ([GH-105746](https://github.com/godotengine/godot/pull/105746))
+- `set_scope` removed from `JSONRPC`. Manual method registration is now required via `set_method`. ([GH-104890](https://github.com/godotengine/godot/pull/104890))
+
+### Animation
+This might be a little more technical than usual, but the work [Tokage](https://github.com/TokageItLab) put into implementing `BoneConstraint3D` warrants a highlight all the same ([GH-100984](https://github.com/godotengine/godot/pull/100984)). With this new class, users will be able to specify two bones simultaneously, opening the door for more natural movements and poses.
```

Comment:
```suggestion
This might be a little more technical than usual, but the work [Tokage](https://github.com/TokageItLab) put into implementing `BoneConstraint3D` warrants a highlight all the same ([GH-100984](https://github.com/godotengine/godot/pull/100984)). With this new class, users will be able to bind bone to another bone, opening the door for more natural movements and poses.
```

Considering that there is a clear master-servant relationship, other terms may be more appropriate.